bin/desi_tsnr_afterburner

```diff
@@ -908,14 +942,30 @@ def main():
             log.warning("Couldn't find the FRAMES HDU, looking for old format name TSNR2_FRAME")
             preexisting_tsnr2_frame_table = read_table(args.outfile,"TSNR2_FRAME")
 
-
-
     # starting computing
     # one night at a time
 
     summary_rows  = list()
 
     def one_night(count, night, rows, tmpfilename):
+        """Prepare multiprocessing steps for analyzing one night.
```

Comment:
If I understand correctly, this function is more than just preparing steps, it is actually running the TSNR calculations for this night.  If that is correct, let's update this docstring description.

Reply:
OK, will do.

Reply:
On further review, technically all `one_night` does is prepare the inputs to `func`, which performs the actual calculation.